doc/source/user/tutorial-svd.rst

```diff
@@ -0,0 +1,468 @@
+Tutorial: Linear algebra on n-dimensional arrays
+================================================
+
+.. currentmodule:: numpy
+
+.. testsetup::
+
+   import numpy as np
+   np.random.seed(1)
+
+Prerequisites
+=============
+
+Before reading this tutorial you should know a bit of Python. If you
+would like to refresh your memory, take a look at the `Python
+tutorial <https://docs.python.org/tutorial/>`__.
+
+If you wish to work the examples in this tutorial, you must also have
+some software installed on your computer. Please see
+https://scipy.org/install.html for instructions.
+
+Learner profile
+===============
+
+This tutorial is aimed at people who have some basic prior knowledge of linear
+algebra and arrays in NumPy and want to understand how n-dimensional
+(:math:`n>=2`) arrays are represented and can be manipulated. In particular, if
+you don't know how to apply common functions to n-dimensional arrays in stacked
+format (without using for-loops), or if you wish to understand axis and shape
+properties for n-dimensional arrays, this tutorial might be of help.
+
+Learning Objectives
+===================
+
+After this tutorial, you should be able to:
+
+- Understand the difference between one-, two- and n-dimensional arrays in NumPy;
+- Understand how to apply some linear algebra operations in stacked format;
+- Understand axis and shape properties for n-dimensional arrays.
+
+Content
+=======
+
+In this tutorial, we will use linear algebra to generate an approximation of an
+image. We'll use the ``face`` image from the `scipy.misc` module:
+
+    >>> from scipy import misc
+    >>> img = misc.face()
+
+If you would rather use your own image, in order to transform this image into a
+numpy array that can be manipulated, we will use the ``imread`` function from the
+`matplotlib.image` submodule.
+
+Now, ``img`` is a numpy array, as we can see when using the ``type`` function:
+
+::
+
+    >>> type(img)
+    numpy.ndarray
+
+We can see the image using the `matplotlib.pyplot.imshow` function:
+
+    >>> import matplotlib.pyplot as plt
+    >>> plt.imshow(img)
+
+.. image:: https://docs.scipy.org/doc/scipy/reference/_images/scipy-misc-face-1.png
+
+Shape, axis and array properties
+--------------------------------
+
+One particular important observation to be made is that, if you are at all
+familiar with linear algebra, you might expect `dimensions` to refer to the
+number of coordinates in each axis of your coordinate system; thus, a
+two-dimensional vector would be a list of two coordinates. In NumPy, however,
+a `dimension` is one of the axes in the array, so a two-dimensional vector
+is the equivalent of a matrix.
+
+First, let's check for the shape of the data in our array. Since this image is
+a two-dimensional set of data (the pixels in the image form a rectangle), we
+might expect a two-dimensional array to represent it (a matrix). However, using
+the ``shape`` property of this NumPy array gives us a different result:
+
+::
```

Comment:
Nit: You can combine these three lines into one if you replace `result:` with `result::`.